sklearn/neighbors/_binary_tree.pxi

```diff
@@ -550,21 +547,23 @@ cdef class NeighborsHeap:
     cdef int _push(self, ITYPE_t row, DTYPE_t val,
                    ITYPE_t i_val) nogil except -1:
         """push (val, i_val) into the given row"""
-        cdef:
-            ITYPE_t size = self.distances.shape[1]
-            DTYPE_t* dist_arr = &self.distances[row, 0]
-            ITYPE_t* ind_arr = &self.indices[row, 0]
-        return heap_push(dist_arr, ind_arr, size, val, i_val)
+        return heap_push(
+            values=&self.distances[row, 0],
+            indices-&self.indices[row, 0],
```

Comment:
My bad.
```suggestion
            indices=&self.indices[row, 0],
```

Reply:
I think this is what is breaking the CI.